src/itinerary.js

```diff
@@ -0,0 +1,6 @@
+function Itinerary(ports) {
+this.ports = ports;
```

Comment:
Indentation should be uniform across all files, since most of them are at 4 spaces (except for `ports.test`) Im assuming thats the intention:
```suggestion
    this.ports = ports;
```